app/views/hosts/console/vnc.html.erb

```diff
@@ -2,7 +2,7 @@
                            "noVNC/des", "noVNC/input", "noVNC/display", "noVNC/rfb",
                            "noVNC/jsunzip", "noVNC/ui", "noVNC") %>
 
-<% title "#{@console[:name]}" %>
+<% title "#{@console[:name]} (pass: #{@console[:password]})" %>
```

Comment:
Sorry but this is way too obvious. Anyone overlooking someone else's screen would see the password (speaking from experience.. Compute Resource passwords were this obvious in the past). 

You can use the JS in here to display it: https://github.com/theforeman/foreman/blob/develop/app/assets/javascripts/hidden_values.js

I'd probably add another button to the button_group to show the password there, title's font is too large. Maybe something like ```Show password + glyphicon-search``` when hidden, and ```#{password} + glyphicon-lock``` when hidden. 

Only make the icon clickable if you follow this approach, otherwise we're going to get an issue open right away saying they can't copy the password because it hides itself when clicked.